### ISSUE TITLE
-B documentation different than template

### ISSUE THREAD

**Issue by folofjc** [jgm/pandoc#7877]
**Explain the problem.**
In the documentation, under "General Writer Options", the option `-B FILE, --include-before-body=FILE|URL` states that "Include contents of FILE, verbatim, at the beginning of the document body (e.g. after the <body> tag in HTML, or the \begin{document} command in LaTeX)."

However, it is actually added after the `frontmatter`, `maketitle`, and abstract in the `LaTeX` template. Therefore these things cannot be changed using this option. 

I would prefer that this option puts the file _before_ the `frontmatter`, abstract, and `maketitle`. However, if there is a reason that it needs to be after these, then the documentation should be updated to reflect this.

**Pandoc version?**
pandoc 2.11, ubuntu 18.04 LTS

**Comment by jgm** [comment]
Thanks for pointing this out.
I see that in the HTML template, include-before goes before the title and abstract.
On the other hand, in Docbook, it goes after the info element (which is probably the only sensible place for it).

Not sure what to do:  the behavior isn't entirely consistent across formats.  One is tempted to make it consistent, but changing this is almost guaranteed to break something for someone!

**Comment by folofjc** [comment]
Yeah, I agree. I can't think of a reason that it should be after the astract, title, and front matter in LaTeX. Those things don't do anything that would make sense for the `-B` option to need to be after them.
